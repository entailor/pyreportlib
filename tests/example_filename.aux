\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\gdef \LT@i {\LT@entry 
    {4}{43.94458pt}\LT@entry 
    {4}{49.77777pt}\LT@entry 
    {4}{49.77777pt}\LT@entry 
    {4}{49.77777pt}\LT@entry 
    {4}{49.77777pt}\LT@entry 
    {4}{49.77777pt}}
\gdef \LT@ii {\LT@entry 
    {4}{43.94458pt}\LT@entry 
    {4}{49.77777pt}\LT@entry 
    {4}{49.77777pt}\LT@entry 
    {4}{49.77777pt}\LT@entry 
    {4}{49.77777pt}\LT@entry 
    {4}{49.77777pt}}
\@writefile{toc}{\contentsline {section}{\numberline {1}Summary}{3}{section.1}\protected@file@percent }
\newlabel{sec:Summary}{{1}{3}{Summary}{section.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Introduction}{3}{section.2}\protected@file@percent }
\newlabel{sec:Introduction}{{2}{3}{Introduction}{section.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Results}{3}{section.3}\protected@file@percent }
\newlabel{sec:Results}{{3}{3}{Results}{section.3}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.1}Plots}{4}{subsection.3.1}\protected@file@percent }
\newlabel{subsec:Plots}{{3.1}{4}{Plots}{subsection.3.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.2}Subsection with fancy latex code}{7}{subsection.3.2}\protected@file@percent }
\newlabel{subsec:Subsectionwithfancylatexcode}{{3.2}{7}{Subsection with fancy latex code}{subsection.3.2}{}}
\newlabel{sec:Heavestroke3hmax}{{3.2}{8}{Subsection with fancy latex code}{subsection.3.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {4}Conclusion}{8}{section.4}\protected@file@percent }
\newlabel{sec:Conclusion}{{4}{8}{Conclusion}{section.4}{}}
\newlabel{LastPage}{{}{8}{}{page.8}{}}
\xdef\lastpage@lastpage{8}
\xdef\lastpage@lastpageHy{8}
