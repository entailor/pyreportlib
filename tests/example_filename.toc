\contentsline {section}{\numberline {1}Summary}{3}{section.1}% 
\contentsline {section}{\numberline {2}Introduction}{3}{section.2}% 
\contentsline {section}{\numberline {3}Results}{3}{section.3}% 
\contentsline {subsection}{\numberline {3.1}Plots}{4}{subsection.3.1}% 
\contentsline {subsection}{\numberline {3.2}Subsection with fancy latex code}{7}{subsection.3.2}% 
\contentsline {section}{\numberline {4}Conclusion}{8}{section.4}% 
