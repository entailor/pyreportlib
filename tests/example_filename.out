\BOOKMARK [1][-]{section.1}{Summary}{}% 1
\BOOKMARK [1][-]{section.2}{Introduction}{}% 2
\BOOKMARK [1][-]{section.3}{Results}{}% 3
\BOOKMARK [2][-]{subsection.3.1}{Plots}{section.3}% 4
\BOOKMARK [2][-]{subsection.3.2}{Subsection with fancy latex code}{section.3}% 5
\BOOKMARK [1][-]{section.4}{Conclusion}{}% 6
